model/reliability/query_test.go

```diff
@@ -0,0 +1,16 @@
+package reliability
+
+import (
+	"testing"
+
+	_ "github.com/evergreen-ci/evergreen/testutil"
+	"github.com/stretchr/testify/suite"
+)
+
+type taskReliabilityQuerySuite struct {
+	suite.Suite
+}
+
+func TestStatsQuerySuite(t *testing.T) {
+	suite.Run(t, new(taskReliabilityQuerySuite))
+}
```

Comment:
I'd prefer using t.Run() for subtests rather that suites, these days (they're a bit easier to track.)

Reply:
I've deleted this file for the moment as there is nothing to test.

I've updated reliability_test.go to remove the suite, but there are no subtests at the moment.

Do you have an example that I should copy? A large chunk of the code seems to use the suite approach. I just copied  test / task stats.